backend/e4h-services/ingestion-service/app/api/endpoints/template_generation.py

```
@@ -72,6 +73,59 @@ async def get_facility_ingestion_template(
         logger.error(f"Unhandled error in get_facility_ingestion_template: {e}")
         raise HTTPException(status_code=500, detail=f"An unexpected error occurred: {str(e)}")
 
+@router.get('/facilityWithStaff',
+            summary='Generate facility ingestion template with staff Excel file',
+            response_description="Returns Excel template with facility schema")
+async def get_facility_ingestion_template_with_staff(
```

Pylint (3.3.7)

[refactor] Too many local variables (18/15)

(R0914)